HealthcareSystem/PatientWebApp/Controllers/EventController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PatientWebApp.Auth;
+using PatientWebApp.Controllers.Adapter;
+using PatientWebApp.DTOs;
+using RestSharp;
+
+namespace PatientWebApp.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EventController : ControllerBase
+    {
+
+        [Authorize(Roles = UserRoles.Patient)]
+        [HttpGet]
+        public ActionResult Add(StatisticEventDTO statisticEventDTO)
+        {
+            return RequestAdapter.SendRequestWithBody("http://localhost:58828", "/api/event", statisticEventDTO);
```

Comment:
Could you configure this like the other service links are configured?

Reply:
Actually, ignore this. I'll configure it, I just realized EventSourcingService wasn't added to the docker files so I'm making a branch for that right now anyway.